pkg/apis/extensions/validation/network_test.go

```diff
@@ -162,17 +162,6 @@ var _ = Describe("Network validation tests", func() {
 					"Field": Equal("spec.serviceCIDR"),
 				}))))
 			})
-
-			It("should forbid Network with overlapping pod and service CIDRs", func() {
```

Comment:
Add a test case that overlapping ranges are allowed for IPv6 (validation is passing for them)?

/hold